test/controllers/beneficial.owner.individual.controller.spec.ts

```diff
@@ -98,6 +109,15 @@ describe("BENEFICIAL OWNER INDIVIDUAL controller", () => {
       expect(resp.header.location).toEqual(BENEFICIAL_OWNER_TYPE_URL);
     });
 
+    test(`redirect to the ${BENEFICIAL_OWNER_TYPE_URL} page after a successful post from ${BENEFICIAL_OWNER_INDIVIDUAL_PAGE} page with service address data`, async () => {
```

Comment:
I'm not sure this is testing what the story is saying - it's talking about choosing the 'yes' then 'no' address radio buttons and ensuring that any address data entered is removed from the session.

I'm therefore wondering if the implementation is delivering what the story describes?

Reply:
PR Updated - Matt and I have been looking at the test and have now added checks to check that mapFieldsToDataObject function is called with service address or not, and made the mapFieldsToDataObject mock return some dummy data so we can test if the service address gets replaced with dummy data during the mapping or if it is replaced with empty object (when they have selected service address is same as residential)